scripts/delete_files_with_no_metadata/delete_files_with_no_metadata.py

```diff
@@ -0,0 +1,107 @@
+import json
+import time
+
+import requests
+import subprocess
+
+DRY_RUN = False
+TOKEN = 'insert-token-without-bearer-prefix'
+SUBMISSION_URL = 'https://api.ingest.archive.data.humancellatlas.org/submissionEnvelopes/insert-submission-id'
+SUBMISSION_UUID = 'insert-submission-uuid'
+FILES_TO_DELETE_FILENAME = f'{SUBMISSION_UUID}_files_to_delete.json'
+
+
+def run(command: str, input: str = None, verbose: bool = True):
+    parsed_command = command.split(' ')
+    proc = subprocess.Popen(parsed_command,
+                            stdin=subprocess.PIPE,
+                            stdout=subprocess.PIPE,
+                            stderr=subprocess.PIPE,
+                            )
+    if input:
+        stdout, stderr = proc.communicate(input.encode())
+    else:
+        stdout, stderr = proc.communicate()
+
+    if verbose:
+        print('$ ' + command)
+        print(stdout.decode())
+        print(stderr.decode())
+
+    return proc.returncode, stdout.decode(), stderr.decode()
+
+
+def remove_file(s3_file_path):
+    if DRY_RUN:
+        command = f'echo "Removing file {s3_file_path}"'
+    else:
+        command = f'aws s3 rm {s3_file_path} --profile embl-ebi'
+
+    exit_code, output, error = run(command)
+
+    if exit_code == 0:
+        return f'Removed {s3_file_path}'
+    else:
+        return error
+
+
+def get_draft_files_in_submission(submission_url):
+    api_base = submission_url.split("/")[2]
+    draft_url = f"https://{api_base}/files/search/findBySubmissionEnvelopeAndValidationState?envelopeUri={submission_url}&state=DRAFT&size=200"
+    r = requests.get(draft_url)
+    r.raise_for_status()
+    draft_files = r.json()["_embedded"]["files"]
+    return draft_files
+
+
+def generate_input_file(submission_url, filename):
+    files_to_delete = get_draft_files_in_submission(submission_url)
+    with open(filename, 'w') as outfile:
+        print(f'Saving info on files to delete to {filename}')
+        json.dump(files_to_delete, outfile, indent=4)
+
+
+if __name__ == '__main__':
+    headers = {
+        'Content-type': 'application/json',
+        'Authorization': f'Bearer {TOKEN}'
+    }
+    # 1. Get the list of file resources to be deleted - save them in a file
+    generate_input_file(submission_url=SUBMISSION_URL, filename=FILES_TO_DELETE_FILENAME)
+
+    with open(FILES_TO_DELETE_FILENAME, 'r') as infile:
```

Comment:
we don't need these 2 lines if you return the `files_to_delete` from the `generate_input_file` method

Reply:
Ah i wanted to keep note of the file metadata we wanted to delete, in case some error happens in between. But you are right, i guess i don't need to load them again. I will fix this.

Reply:
I changed my mind, I think it's better to load them from the file so that if the script has to be rerun, we just need to comment out the request to get files.